README.md

```diff
@@ -79,19 +79,19 @@ ninja install
 
 ## Using VKD3D
 
-VKD3D can be used by projects that target Direct3D 12 as a drop-in replacement
-at build-time with some modest source modifications.
+VKD3D can be used by projects targeting Direct3D 12 as an alternative
+at the time of construction with some modest source changes.
 
-If VKD3D is available when building Wine, then Wine will use it to support
-Direct3D 12 applications.
+If VKD3D is available when building Wine, Wine will use it for support
+Direct3D Applications 12.
 
 ## Environment variables
 
-Most of the environment variables used by VKD3D are for debugging purposes. The
-environment variables are not considered a part of API and might be changed or
-removed in the future versions of VKD3D.
+Most of the environment variables used by VKD3D are debugged. The
+environment variables are not part of the API and can be changed or
```

Comment:
We're not talking about whether they are *part of* the API, we're simply talking about whether they are *considered to be* part of the API.